Doc/source/release_notes.rst

```diff
@@ -50,6 +50,14 @@ required to use it. These dependencies were removed in SpacePy 0.3.0, but
 were still required for use of ``LANLstar``. Support for ``LANLstar`` is
 reinstated in SpacePy 0.4.0.
 
+Deprecations and removals
+*************************
+:func:`~spacepy.irbempy.get_sysaxes`, :func:`~spacepy.irbempy.sph2car`,
+and :func:`~spacepy.irbempy.car2sph` were deprecated in SpacePy 0.2.2
+and have been removed. In place of the latter functions,
+:func:`~spacepy.coordinates.sph2car` and
+:func:`~spacepy.coordinates.sph2car` should be used.
+
```

Comment:
- Not entirely sure how I missed the earlier deprecations and removals section...
- Yes
- get_sysaxes is included as being deprecated and removed in the first line. Given its very niche function I'm not sure it needs explicit documentation for how to replace.
- Good catch

Reply:
Reading comprehension fail on my part, sorry.

I also realized one more thing...this will basically render as "car2sph was removed, use car2sph instead". Would it make sense to do something like ":mod:`~spacepy.irbempy` functions ``get_sysaxes``, ``sph2car``, and ``car2sph`` were deprecated...."?